source/_integrations/traccar_server.markdown

```diff
@@ -127,6 +116,120 @@ Tracker:
 
 {% enddetails %}
 
+### Sensor - Address
+
+The Traccar Server integration will create a [sensor](/integrations/sensor) entity for each device registered in Traccar Server to show the address reported by the Traccar Server.
+
+This entity is disabled by default.
+
+{% configuration_basic %}
+Name:
+  description: The name of the sensor will be set to what you have named it in Traccar Server followed by Address.
+Entity ID:
+  description: This will be a slugified version of the name.
+Unique ID:
+  description: This will be the unique ID of the device tracker in Traccar Server followed by `position_address`.
+State:
+  description: This will be the address reported by the Traccar Server, if geo detection is not configured this will be unknown`.
```

Comment:
```suggestion
  description: This will be the address reported by the Traccar Server, if geofence is not configured, this will be unknown`.
```
Should this be geofence? This is the only occurrence of geo detection I find here. or is this something else?

Reply:
Geo detection is a setting that needs to be enabled on the server.